src/rosdiscover/recover/loader.py

```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
 from __future__ import annotations
 
+from .symbolic import NegateExpr
```

Comment:
I'm really confused as to why the CI isn't flagging this as an issue 😕 Does tox not point this out when you run locally? In any case: this import should go onto the appropriate location in the import starting on line 28

Reply:
fixed